ui/view.go

```diff
@@ -230,7 +230,6 @@ func (v *View) showKindPage(k Kind) error {
 // Go back page based on current kind
 func (v *View) back() {
 	toPage := v.kind.prevKind().getAppPageName(v.getClusterArn())
-	v.kind = v.kind.prevKind()
```

Comment:
☝️ A callout here: Without removing this, my implementation was somehow vulnerable to the following panic when using the back arrow to navigate to the previous page:

```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
        panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x40 pc=0x100fc8a50]

goroutine 1 [running]:
github.com/rivo/tview.(*Application).Run.func1()
        /Users/mike.ball/go/pkg/mod/github.com/rivo/tview@v0.0.0-20230525073430-4a1f85bb2219/application.go:251 +0x4c
panic({0x1010f6b80?, 0x10148ec90?})
        /opt/homebrew/Cellar/go/1.21.5/libexec/src/runtime/panic.go:914 +0x218
github.com/keidarcy/e1s/ui.(*View).changeSelectedValues(0x140002ae1c0)
        /Users/mike.ball/go/src/github.com/keidarcy/e1s/ui/table.go:159 +0xd0
github.com/keidarcy/e1s/ui.(*View).handleSelectionChanged(0x140002ae1c0, 0x5?, 0x140004ddc20?)
        /Users/mike.ball/go/src/github.com/keidarcy/e1s/ui/table.go:75 +0x28
github.com/rivo/tview.(*Table).InputHandler.func1(0x140006900e0, 0x1015e2b38?)
        /Users/mike.ball/go/pkg/mod/github.com/rivo/tview@v0.0.0-20230525073430-4a1f85bb2219/table.go:1612 +0x4a8
github.com/rivo/tview.(*Table).InputHandler.(*Box).WrapInputHandler.func2(0x140004ddc20?, 0x100fae170?)
        /Users/mike.ball/go/pkg/mod/github.com/rivo/tview@v0.0.0-20230525073430-4a1f85bb2219/box.go:167 +0x54
github.com/rivo/tview.(*Pages).InputHandler.func1(0x140035f5c50?, 0x0?)
        /Users/mike.ball/go/pkg/mod/github.com/rivo/tview@v0.0.0-20230525073430-4a1f85bb2219/pages.go:311 +0xa8
github.com/rivo/tview.(*Pages).InputHandler.(*Box).WrapInputHandler.func2(0x140023beae0?, 0x100fa4f40?)
        /Users/mike.ball/go/pkg/mod/github.com/rivo/tview@v0.0.0-20230525073430-4a1f85bb2219/box.go:167 +0x54
github.com/rivo/tview.(*Flex).InputHandler.func1(0x140023beae0?, 0x1?)
        /Users/mike.ball/go/pkg/mod/github.com/rivo/tview@v0.0.0-20230525073430-4a1f85bb2219/flex.go:251 +0xd0
github.com/rivo/tview.(*Flex).InputHandler.(*Box).WrapInputHandler.func2(0x140036008d0?, 0xe170?)
        /Users/mike.ball/go/pkg/mod/github.com/rivo/tview@v0.0.0-20230525073430-4a1f85bb2219/box.go:167 +0x54
github.com/rivo/tview.(*Pages).InputHandler.func1(0x10?, 0x101118dc0?)
        /Users/mike.ball/go/pkg/mod/github.com/rivo/tview@v0.0.0-20230525073430-4a1f85bb2219/pages.go:311 +0xa8
github.com/rivo/tview.(*Pages).InputHandler.(*Box).WrapInputHandler.func2(0x14000159ad0?, 0x1400360dd68?)
        /Users/mike.ball/go/pkg/mod/github.com/rivo/tview@v0.0.0-20230525073430-4a1f85bb2219/box.go:167 +0x54
github.com/rivo/tview.(*Application).Run(0x14000330000)
        /Users/mike.ball/go/pkg/mod/github.com/rivo/tview@v0.0.0-202305
25073430-4a1f85bb2219/application.go:344 +0x4a8
github.com/keidarcy/e1s/ui.Show()
        /Users/mike.ball/go/src/github.com/keidarcy/e1s/ui/app.go:92 +0xd8
main.main()
        /Users/mike.ball/go/src/github.com/keidarcy/e1s/main.go:12 +0x1c
```